api/src/main/java/org/pmiops/workbench/db/dao/DataSetServiceImpl.java

```diff
@@ -591,6 +590,16 @@ private ValuesLinkingPair getValueSelectsAndJoins(List<DomainValuePair> domainVa
     return new ValuesLinkingPair(valueSelects, valueJoins);
   }
 
+  private static String toTitleCase(String name) {
+    if (name.isEmpty()) {
+      return name;
+    } else if (name.length() == 1) {
+      return name;
```

Comment:
Why would it be correct to leave a single letter uncapitalized here?

Reply:
Good catch. We don't have any single-letter names at the moment.